### PR TITLE
Addresses #167

### PR DESCRIPTION
**Pull request by joseph-robertson**
https://github.com/NREL/OpenStudio-ERI/issues/167

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -79,4 +80,1837 @@ task :update_measures do
...
+    if File.exists? File.join(tests_dir, derivative)
+      old_hpxml_doc = XMLHelper.parse_file(File.join(tests_dir, derivative))
+      created_date_and_time = HPXML.get_hpxml_values(hpxml: old_hpxml_doc.elements["HPXML"])[:created_date_and_time]
+      hpxml.elements["XMLTransactionHeaderInformation/CreatedDateAndTime"].text = created_date_and_time
+    end
+
+    HPXML.add_site(hpxml: hpxml, **site_values) unless site_values.nil?
+    building_occupancys_values.each do |building_occupancy_values|
+      HPXML.add_building_occupancy(hpxml: hpxml, **building_occupancy_values)
+    end
```
Comment (shorowit) [inline Rakefile]: Hmm, why are we looping here? There can only be a single occupancy value.

Comment (shorowit) [reply Rakefile]: I'm going to make a couple small changes so I'll change this.

Comment (joseph-robertson) [reply Rakefile]: Should probably change here, too? https://github.com/NREL/OpenStudio-HPXML/blob/master/Rakefile#L409-L411

**Comment by joseph-robertson** [comment]
Actually, do we even need to return the hashes or arrays (e.g., `building_occupancy_values`) from their respective methods since they are initialized outside the loop?

**Comment by shorowit** [comment]
In the case of `building_occupancy_values`, the method is returning a hash in order to override what was set by the parent HPXML file.  I'll check the various methods to see if it's needed though.

**Pull request merged**